eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -562,7 +567,7 @@ object Helpers {
       * @param tx           the remote commitment transaction that has just been published
       * @return a list of transactions (one per HTLC that we can claim)
       */
-    def claimRemoteCommitTxOutputs(keyManager: KeyManager, commitments: Commitments, remoteCommit: RemoteCommit, tx: Transaction)(implicit log: LoggingAdapter): RemoteCommitPublished = {
+    def claimRemoteCommitTxOutputs(feeTargets: FeeTargets, keyManager: KeyManager, commitments: Commitments, remoteCommit: RemoteCommit, tx: Transaction, feeEstimator: FeeEstimator)(implicit log: LoggingAdapter): RemoteCommitPublished = {
```

Comment:
`feeTargets` should be after `feeEstimator`

Reply:
Agree, done in f0649dd